staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -77,6 +80,7 @@ func NewCommandStartAggregator(defaults *AggregatorOptions, stopCh <-chan struct
 
 // AddFlags is necessary because hyperkube doesn't work using cobra, so we have to have different registration and execution paths
 func (o *AggregatorOptions) AddFlags(fs *pflag.FlagSet) {
+	o.ServerRunOptions.AddUniversalFlags(fs)
```

Comment:
Won't this double-register the flags in the kube-apiserver binary?

Reply:
Or, wait, maybe this is correct, I need to double check which layer kube-apiserver imports

Reply:
It won't. The AggregatorOptions is used when the kube-aggregator starts standalone.